eni.tf

```diff
@@ -0,0 +1,31 @@
+locals {
+  additional_ips_count = "${var.associate_public_ip_address && var.instance_enabled && var.additional_ips_count != "0" ? var.additional_ips_count : 0}"
+}
+
+resource "aws_network_interface" "additional" {
+  count     = "${local.additional_ips_count}"
+  subnet_id = "${var.subnet}"
+
+  security_groups = [
+    "${compact(concat(list(var.create_default_security_group ? join("", aws_security_group.default.*.id) : ""), var.security_groups))}",
+  ]
+
+  tags {
```

Comment:
this should be `tags = "${module.label.tags}"`

Reply:
will be fixed in https://github.com/cloudposse/terraform-aws-ec2-instance/pull/15